rst2pdf/tests/input/test_multiple_styles.rst

```diff
@@ -0,0 +1,41 @@
+Multiple styles test
```

Comment:
Hurrah for tests and tests that we can tell from the output what was supposed to happen :) Are there some corner cases where there could be stylesheet styles on an element because of its type, plus a classname? I think more tests could be useful.